src/System.IO.Compression/ref/System.IO.Compression.cs

```diff
@@ -93,6 +93,7 @@ public partial class ZipArchiveEntry
         public System.DateTimeOffset LastWriteTime { get { throw null; } set { } }
         public long Length { get { throw null; } }
         public string Name { get { throw null; } }
+        public uint Crc32 { get { throw null; } }
```

Comment:
This will need to be marked as [CLSCompliant(false)].  We should do the same in the src file as well.

Reply:
ty :)